aries_cloudagent/core/conductor.py

```diff
@@ -592,6 +592,26 @@ async def outbound_message_router(
         """
         Route an outbound message.
 
+        Args:
+            profile: The active profile for the request
+            message: An outbound message to be sent
+            inbound: The inbound message that produced this response, if available
+        """
+        status: OutboundSendStatus = await self._outbound_message_router(
+            profile=profile, outbound=outbound, inbound=inbound
+        )
+        await profile.notify(status.topic, outbound)
+        return status
```

Comment:
I chose to wrap the `outbound_message_router` to clean up the event emission, as that felt easiest to implement. It's not my favorite adding in more nested methods, but it seemed simple enough and avoided heavy modifications to the existing method. 
If you prefer the other method you suggested, we can go back and change it; I don't see much difference either way, other than a few minutes more spent writing code.